packages/scaffold/src/views/w3m-swap-preview-view/index.ts

```diff
@@ -95,6 +103,12 @@ export class W3mSwapPreviewView extends LitElement {
   }
 
   // -- Private ------------------------------------------- //
+  private refreshTransaction() {
+    this.interval = setInterval(() => {
+      SwapController.getTransaction()
+    }, 10_000)
```

Comment:
what's the ttl on the api side?

Reply:
Would be nice to validate with Max for best number to do this. This is also a UX question. The user should get the up-to-date swap values (token prices, quote, gas price etc.) periodically, I'm not sure if longer duration would create bad UX